firstvoices/backend/tests/test_apis/test_import_job_api.py

```diff
@@ -288,7 +282,6 @@ def test_detail_superadmin_access(self, visibility):
         assert response.status_code == 200
 
     @pytest.mark.parametrize("role", [Role.MEMBER, Role.ASSISTANT])
-    @pytest.mark.django_db
```

Comment:
Interesting that these tests don't need to be marked despite creating things in the db. Any reasoning behind these removals? Just curious I'm okay with it either way.

Reply:
Yes, I figured since all the tests need db access, I moved this decorator to the top of the class, which gives all the methods inside the class db access.